data/sql_updates/post_totals_create_candidate_aggregates.sql

```diff
@@ -51,7 +55,9 @@ election_aggregates as (
         candidate_id,
         election_year,
         sum(receipts) as receipts,
-        sum(disbursements) as disbursements
+        sum(disbursements) as disbursements,
+        sum(receipts) >= 5000 as five_thousand_flag,
+        array_agg(federal_funds_flag) @> array[cast('true' as boolean)] as federal_funds_flag
```

Comment:
Same question as above--do we need the `cast` here? How about something like `true != any array_agg(federal_funds_flag)`?